ee/clickhouse/views/experiments.py

```diff
@@ -47,6 +47,9 @@ class Meta:
         ]
 
     def validate_feature_flag_key(self, value):
+        if self.context['request'].method == 'PUT' or self.context['request'].method == 'PATCH' and self.initial_data["feature_flag_key"] == value:
```

Comment:
maybe there's a better way to check for this? 😕 @neilkakkar

Reply:
what's the issue? O.o

Reply:
ah, I think I see it: instead of patch, we send `create` request when trying to save an edited experiment?

Reply:
Actually, after the serializer update I made in https://github.com/PostHog/posthog/pull/7981, we don't really need this validation here at all.